tests/test_e2e.py

```diff
@@ -0,0 +1,87 @@
+import pytest
+
+from neptune_prophet.impl import (
+    create_forecast_plots,
+    create_residual_diagnostics_plots,
+    create_summary,
+    get_forecast_components,
+    get_model_config,
+    get_serialized_model,
+)
+
+try:
+    # neptune-client=0.9.0+ package structure
+    import neptune.new as neptune
+except ImportError:
+    # neptune-client>=1.0.0 package structure
+    import neptune
+
+
+@pytest.mark.parametrize("log_interactive", [False, True])
+def test_e2e(dataset, model, log_interactive):
+
+    if log_interactive:
+        try:
+            import plotly  # pylint: disable=import-outside-toplevel, unused-import
+        except ModuleNotFoundError:
+            pytest.skip("plotly is needed for log_interactive to work")
```

Comment:
We should add `plotly` to test requirements and omit this check. In Checks tab you can see that for `log_interactive=True` this test will be just skipped.

Reply:
Plotly added to dependencies. I'll keep the `try ... except ...` block because it is useful for testing locally, also w/o plotly.

Reply:
It's now in `dev`/test requirements so we can assume that developer will have `plotly` locally :wink:

Reply:
But it's not in the main requirements, so like this, I leave the possibility to at least test it manually in the scenario w/o plotly. AFAIK it's not possible to have an optional dependency in pytest otherwise. It is probably also not that important for us to have a more sophisticated solution right now.